test/tests/functional/pbs_trillion_jobid.py

```diff
@@ -444,3 +444,27 @@ def test_jobid_duplication(self):
         # Job should submit successfully because all existing id's has been
         # passed
         self.submit_job(lower=1, upper=2, job_id='3[]')
+
+    def test_jobid_resvid_after_multiple_restart(self):
+        """
+        Test to check the Jobid/Resvid should not wrap to 0 during
+        server restart multiple times consecutively either gracefully/abruptly
+        """
+        a = {'resources_available.ncpus': 15}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
```

Comment:
Why do you need to set node's resources available? You do not really care if the job is running or not.

Reply:
Using a generic function self.submit_job() in which we were checking the R state, however have updated that as well.